Gruntfile.js

```diff
@@ -1,8 +1,24 @@
+/**
+ * Gruntfile.
+ * @type {[type]}
+ */
 module.exports = function(grunt) {
+
+  /**
+   * [sass description]
+   * @type {[type]}
+   */
+  const sass = require('node-sass');
```

Comment:
New version of grunt-sass needs this.
FYI:
https://davidgracieweb.co.uk/node-sass-vs-grunt-sass-vs-dart-sass-vs-ruby-sass/